resolver_plugin.go

```diff
@@ -86,11 +86,17 @@ func (m *ResolverPlugin) generateSingleFile(data *codegen.Data, models []*Model,
 		Alias:      "dm",
 		ImportPath: path.Join(m.rootImportPath, m.backend.Directory),
 	})
+
 	file.Imports = append(file.Imports, Import{
 		Alias:      "fm",
 		ImportPath: path.Join(m.rootImportPath, m.frontend.Directory),
 	})
 
+	file.Imports = append(file.Imports, Import{
```

Comment:
This is not needed in my opinion as this is the same as the "fm" directory?

Reply:
The code will not compile without importing the `Resolvers` from generated graphql code.  They are not the same directory in my case.  The fm package contains the graphql model code.  The gm package contains the generated graphql code.

These need to be imported from the generated graphql code.
```
func (r *Resolver) Mutation() gm.MutationResolver	{ return &mutationResolver{r} }
func (r *Resolver) Query() gm.QueryResolver		{ return &queryResolver{r} }

type mutationResolver struct{ *Resolver }
type queryResolver struct{ *Resolver }
```

Perhaps there is an issue with directory configuration/assumptions.